### PR TITLE
Mobile friendly Header

### PR DESCRIPTION
**Pull request by sprice**
This makes the Header mobile friendly.

Of note, address and network details have been moved to the sidebar.

### CODE REVIEW

File: src/components/AppSideBar.vue [hunk 1]
```diff
@@ -4,34 +4,37 @@
       <img src="../assets/logos/codex/gold.svg">
     </div>
     <div class="button-container" v-if="isAuthenticated">
-      <b-link to="/collection">
+      <b-link to="/collection" @click="hideSideNav">
```
Comment (johnhforrest) [inline src/components/AppSideBar.vue]: Any time something is copy/pasted many times it feels like a code smell to me. Can you abstract this out to make it less redundant?

Although this begs the bigger question, why is this needed?

Comment (sprice) [reply src/components/AppSideBar.vue]: Without this, when in a mobile view and the navigation bar is showing, when clicking on a link, you are taken to that page in the background but the nav bar still fills the entire viewport.

**Comment by sprice** [comment]
@Anaphase both those items addressed.

File: src/components/AppSideBar.vue (hunk 1 above)
Comment (sprice) [reply src/components/AppSideBar.vue]: @johnhforrest I removed the multiple `@click`s and replaced with vanilla `addEventListener('click',...`
